meta-mender-commercial/recipes-mender/mender-client/mender-client_%.bbappend

```diff
@@ -0,0 +1 @@
+# Deliberately empty bbappend file to remove bitbake warnings about no bb files
```

Comment:
As pointed out in the master PR (#1060) I don't think we need this file at all. Please remove or prove me wrong :)

Reply:
And follow-up accordingly in zeus (#1057) and warrior (#1058).